cpp/ql/src/experimental/Security/CWE/CWE-1041/FindWrapperFunctions.ql

```diff
@@ -0,0 +1,140 @@
+/**
+ * @name Find Wrapper Functions
+ * @description --Finding for function calls for which wrapper functions exist.
+ * @kind problem
+ * @id cpp/find-wrapper-functions
+ * @problem.severity warning
+ * @precision medium
+ * @tags correctness
```

Comment:
Do you think this query would also fit under the `maintainability` tag? (I'm not saying this shouldn't be under `security` as well)

Reply:
I thought `correctness` was more general and neutral. but if you insist I will add `maintainability`